repos/spack_repo/builtin/packages/scorep/package.py

```diff
@@ -207,6 +207,8 @@ def configure_args(self):
         ]
 
         cname = self.clean_compiler(spec.compiler.name)
+        if cname == "intel-oneapi-compilers":
+            cname = "oneapi"
```

Comment:
Please keep renames together in the `clean_compiler` helper. But the logic here should be fine.

Reply:
Thanks for pointing that out. I'll wait for the change before merging.

Reply:
Done!